frontend/javascripts/messages.tsx

```diff
@@ -436,6 +436,9 @@ instead. Only enable this option if you understand its effect. All layers will n
   "ui.no_form_active": "Could not set the initial form values as the form could not be loaded.",
   "organization.plan.upgrage_request_sent":
     "An email with your upgrade request has been sent to the WEBKNOSSOS sales team.",
-  "organization.plan.feature_not_available": (requiredPlan: string) =>
-    `This feature is not available in your organization's plan. Ask your organization owner to upgrade at least a ${requiredPlan} plan.`,
+  "organization.plan.feature_not_available": (
+    requiredPlan: string,
+    organizationOwnerName: string,
+  ) =>
+    `This feature is not available in your organization's plan. Ask your organization owner ${organizationOwnerName} to upgrade to at least a ${requiredPlan} plan.`,
```

Comment:
"Ask the owner of your organization (John Doe)" would be more idiomatic english than "organization owner", I think.
Also, I'd change the type to `organizationOwnerName: string | null` and then do a conditional on that so that the double space is avoided if the owner is not known.

Reply:
> Also, I'd change the type to organizationOwnerName: string | null and then do a conditional on that so that the double space is avoided if the owner is not known.

Mhm, I was hoping that `getFeatureNotAvailabeInPlanMessage()` already checks against null to avoid doing this logic in the `messages` module. it, does not, however, prevent a double space.

Reply:
ok, i see. i don't even know whether the double space is rendered at all? I think this could get collapsed into a single space by the browser? if it's rendered, fixing the doublespace without moving the null-check could work something like this: `... owner ${organizationOwnerName}to` (no space before "to") and then `organizationOwnerName` should have a trailing space if it's not empty...

Reply:
TBH, I would not optimize for extremely unlikely case that an orga has no owner. This can only ever be the case if a DB migration was not executed correctly.

Reply:
good point :+1: